core-primitives/sgx/crypto/src/traits.rs

```diff
@@ -8,7 +8,7 @@ pub trait StateCrypto {
 }
 
 pub trait ShieldingCrypto {
-	type Error;
+	type Error: Debug;
 	fn encrypt(&self, data: &[u8]) -> Result<Vec<u8>, Self::Error>;
```

Comment:
This caused issues when formatting the errors. And also it's now consistent with `StateCrypto`, which already had this trait bound on its `Error` associated type.